test/test_client.py

```diff
@@ -168,3 +168,45 @@ def test_get_escrow_duration(self):
         client.login(self.credentials.login, self.credentials.password, self.steam_guard_file)
         response = client.get_escrow_duration(sample_trade_url)
         self.assertEqual(response, 15)
+
+    def test_get_all_listings_from_market(self):
+        client = SteamClient(self.credentials.api_key)
+        client.login(self.credentials.login, self.credentials.password, self.steam_guard_file)
+        listings = client.get_my_market_listings()
+        self.assertTrue(len(listings) == 2)
+        self.assertTrue(len(listings.get("buy_orders")) == 1)
+        self.assertTrue(len(listings.get("sell_listings")) == 1)
+        self.assertIsInstance(next(iter(listings.get("sell_listings").values())).get("description"), dict)
+
+    def test_create_and_remove_sell_listing(self):
+        client = SteamClient(self.credentials.api_key)
+        client.login(self.credentials.login, self.credentials.password, self.steam_guard_file)
+        game = GameOptions.DOTA2
+        inventory = client.get_my_inventory(game)
+        asset_id_to_sell = None
+        for asset_id, item in inventory.items():
+            if item.get("marketable") == 1:
+                asset_id_to_sell = asset_id
+                break
+        self.assertIsNotNone(asset_id_to_sell, "You need at least 1 marketable item to pass this test")
+        response = client.create_sell_listing(asset_id_to_sell, game, 10000)
+        self.assertTrue(response["success"])
+        sell_listings = client.get_my_market_listings()["sell_listings"]
+        listing_to_cancel = None
+        for listing in sell_listings.values():
+            if listing["description"]["id"] == asset_id_to_sell:
+                listing_to_cancel = listing["listing_id"]
+                break
+        self.assertIsNotNone(listing_to_cancel)
+        response = client.remove_sell_listing(listing_to_cancel)
+
+    def test_create_and_cancel_buy_order(self):
+        client = SteamClient(self.credentials.api_key)
+        client.login(self.credentials.login, self.credentials.password, self.steam_guard_file)
+        # PUT THE REAL CURRENCY OF YOUR STEAM WALLET, OTHER CURRENCIES WILL NOT WORK
+        response = client.create_buy_order("AK-47 | Redline (Field-Tested)", 10, 2, GameOptions.CS, Currency.EURO)
+        buy_order_id = response["buy_orderid"]
+        self.assertTrue(response["success"])
+        self.assertIsNotNone(buy_order_id)
+        response = client.cancel_buy_order(buy_order_id)
+        self.assertTrue(response["success"])
```

Comment:
This test should works with everyone by changing the currency according to your steam wallet